Tella/Scenes/Settings/Views/Servers/Uwazi/Step3/UwaziLoginView.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  UwaziLoginView.swift
+//  Tella
+//
+//  Created by Robert Shrestha on 4/24/23.
+//  Copyright © 2023 INTERNEWS. All rights reserved.
+//
+
+import SwiftUI
+
+struct UwaziLoginView: View {
+    @EnvironmentObject var serverViewModel : UwaziServerViewModel
+    @EnvironmentObject var serversViewModel : ServersViewModel
+
+    @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
+    @EnvironmentObject var mainAppModel : MainAppModel
+
+    @State var presentingSuccessLoginView : Bool = false
+
+    var body: some View {
+
+        ContainerView {
+            ZStack {
+                VStack(spacing: 0) {
+                    VStack(spacing: 0) {
+                        Spacer()
+                        TopServerView(title: LocalizableSettings.UwaziLoginAccess.localized)
+                        Spacer()
+                            .frame(height: 40)
+                        TextfieldView(fieldContent: $serverViewModel.username,
```

Comment:
@rob-horizontal-iOS Could you put the code of every view into a separate var or function that return the view to have a good code readeability.